_infdemos/ingest-Large-Data.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Ingesting Externally Hosted Large Datasets
+main_author: Amy Paguirigan
+primary_reviewers: vortexing
+---
+
+Before ingesting any external data sets, please refer to the [Overview of Data Storage at Fred Hutch](/computing/store_overview/) page to begin to identify the final home of the data you are ingesting.  Choosing a location will depend on how large the data are, how often you need to access them, and what type of security concerns might apply to the dataset.  
+
+Once you have an idea where the data will need to live at Fred Hutch, you can begin the process of transferring them to Fred Hutch supported data storage systems.  
+
+Ideally, one wouldn't transfer large amounts of data intended to be temporary into systems like *Fast* storage as that data will be included in the frequent snapshots for backup purposes in that storage system.  This means these data will have a large footprint in the overall system, requiring long term backup of what are actually temporary data (ironic in a way). A better way to ingest and store large data sets is to first ingest the data into *Scratch* storage and then move them into secure, stable object storage systems like *Economy* (either *Local* or *Cloud*).  
+
+## FTP to *Scratch*
+First, make a directory in Scratch for the data you want to ingest.  Then use `wget` to copy, in this case recursively, the data from the ftp url (e.g., ftp.broadinstitute.org/bundle/) using the username (user) and password (pass) required.  
+
+```
+cd /fh/scratch/delete10/lastname_f/
+mkdir ingestedDataDir
+cd ingestedDataDir
+wget --recursive ftp://user@ftp.broadinstitute.org/bundle/
```

Comment:
There are ways of _more_ securely (but not completely securely) providing credentials to wget (and curl).  These typically involve an unencrypted file which may contravene some security policies.  I'll see if I can write something for the "helpful wget" section.